### PR TITLE
Historical charts

### PR DESCRIPTION
**Pull request by iwooden**
Adding historical data charts to the web interface.

[Video](https://www.loom.com/share/b1a72b535fbd48909eb02e05a01a7924?sid=a77d5d57-1df5-47de-94ac-3de5dd5001b9)

### CODE REVIEW

File: packages/dev-frontend/src/pages/Charts.tsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import { Chart } from "../components/Dashboard/Chart/Chart";
+import { ChartProvider } from "../components/Dashboard/Chart/context/ChartProvider";
+
+export const Charts = (): JSX.Element => (
+    <Container>
+      <Heading as="h2" sx={{ mt: "2.5em", fontWeight: "semibold" }}>
+        Historical Charts
+      </Heading>
+      <Container variant="dashboardGrid">
+        <ChartProvider dataSource='lastGoodPrice'>
```
Comment (codyborn) [inline packages/dev-frontend/src/pages/Charts.tsx:12]: It'd be nice to also include informational hints that explain what these mean
image

File: packages/dev-frontend/src/config/index.ts [hunk 2]
```diff
@@ -17,11 +15,14 @@ const parseConfig = (json: unknown): ThresholdConfig => {
...
-      console.error("Malformed coingeckoIdsBySymbol:");
+    if (hasKey(json, "historicalApiUrl") && json.historicalApiUrl !== "") {
+      const { historicalApiUrl } = json;
+      if (typeof historicalApiUrl === "string") {
+        config.historicalApiUrl = historicalApiUrl;
+      } else {
+        console.error("Malformed historical API URL:");
+        console.log(historicalApiUrl);
+      }
     }
```
[the remark this answers is not in the document]
Comment (iwooden) [reply packages/dev-frontend/src/config/index.ts:28]: Implemented!

File: packages/dev-frontend/src/pages/Charts.tsx (hunk 1 above)
Comment (iwooden) [reply packages/dev-frontend/src/pages/Charts.tsx]: Added tooltips to charts

**Review by codyborn — APPROVED** [review]
🚢🚢🚢

**Pull request merged**